@here/harp-omv-datasource/test/MapViewPickingTest.ts

```diff
@@ -0,0 +1,326 @@
+                image: {
+                    width: 512,
+                    height: 512
+                }
+            };
+        });
+
+        if (inNodeContext) {
+            const theGlobal: any = global;
+            theGlobal.window = {
+                window: { devicePixelRatio: 1.0 },
+                location: {
+                    href: "http://mocha-test"
+                }
+            };
+            theGlobal.navigator = {};
+            theGlobal.requestAnimationFrame = (cb: () => void) => {
+                return setTimeout(() => cb(), 15);
+            };
+            theGlobal.cancelAnimationFrame = (id: any) => {
+                return clearTimeout(id);
+            };
+            theGlobal.performance = { now: Date.now };
+        }
+
+        addEventListenerSpy = sinon.stub();
+        removeEventListenerSpy = sinon.stub();
+        canvas = ({
+            clientWidth: 800,
+            clientHeight: 600,
+            addEventListener: addEventListenerSpy,
+            removeEventListener: removeEventListenerSpy
+        } as unknown) as HTMLCanvasElement;
+
+        mapView = new MapView({
+            canvas,
+            decoderCount: 0,
+            theme: ThemeLoader.load(THEME),
+            enableRoadPicking: true
+        });
+
+        await waitForEvent(mapView, MapViewEventNames.ThemeLoaded);
+
+        const geoJsonDataProvider = new GeoJsonDataProvider("italy_test", GEOJSON_DATA, {
+            tiler: new GeoJsonTiler()
+        });
+
+        geoJsonDataSource = new OmvDataSource({
+            decoder: new OmvTileDecoder(),
+            dataProvider: geoJsonDataProvider,
+            name: "geojson",
+            styleSetName: "geojson",
+            gatherFeatureIds: true,
+            createTileInfo: true
+        });
+
+        await mapView.addDataSource(geoJsonDataSource);
+    });
+
+    afterEach(function() {
+        sandbox.restore();
+        mapView.dispose();
+        if (inNodeContext) {
+            delete global.window;
+            delete global.requestAnimationFrame;
+            delete global.cancelAnimationFrame;
+            delete global.navigator;
+        }
+    });
+
+    it("Decoded tile is created", async () => {
+        let tile = geoJsonDataSource.getTile(tileKey);
+
+        assert(tile !== undefined);
+        tile = tile as OmvTile;
+
+        assert.notEqual(tile.tileLoader, undefined);
+
+        const state = await tile.tileLoader!.loadAndDecode();
+        assert.equal(state, TileLoaderState.Ready);
+
+        const decodeTile = tile.decodedTile as DecodedTile;
+        assert.notEqual(decodeTile, undefined);
+
+        assert.notEqual(decodeTile.textGeometries, undefined);
+        assert.notEqual(decodeTile.geometries, undefined);
+        assert.equal(decodeTile.geometries.length, 2);
+
+        const tileInfo = decodeTile.tileInfo as ExtendedTileInfo;
+        assert.notEqual(tileInfo, undefined);
+
+        describe("Test decodeTile, tileInfo data", () => {
```

Comment:
There is no output for the test cases in this scope. It's not clear whether do they run or not.